parachain/primitives/router/src/outbound/mod.rs

```diff
@@ -113,7 +113,7 @@ where
 		// validate the message
 		let (ticket, fee) = OutboundQueue::validate(&outbound_message).map_err(|err| {
 			log::error!(target: "xcm::ethereum_blob_exporter", "OutboundQueue validation of message failed. {err:?}");
-			SendError::Unroutable
+			SendError::Fees
```

Comment:
Not sure this is right. The message validation failure could be caused by a payload that is too large, an invalid channel, etc. So maybe `InvalidMessage` or something is better.

Reply:
Nice catch! My first glance `Unroutable` does not make sense so changed to another one from https://github.com/Snowfork/polkadot-sdk/blob/d0e0b386187b5958b0d6e7ae9aadec9e0b88388e/polkadot/xcm/src/v3/traits.rs#L417-L439,  but as you mentioned `Fees` does not cover all the cases you mentioned.

I can't find another error more sensible in the current list, maybe we can add another PR  to `polkadot-sdk` for better error categories but not the scope of this PR so just reverted.